packages/store-sdk/src/storage/useLocalStorage.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Safe localstorage interface. These try..catch are usefull because
+ * some browsers may block accesss to these APIs due to security policies
+ *
+ * Also, the local storage value is lazy-loaded to avoid hydration mismatch
+ * between server/browser. When state is 'hydrated', the in the heap is the
```

Comment:
What did you mean here?

```suggestion
 * between server/browser. When state is 'hydrated', the in value the heap is the
```

Reply:
`the value in the heap`